tests/test_fit_result.py

```diff
@@ -68,27 +79,88 @@ def test_no_constraint(minimizer, fit_kwargs, expected_par, expected_std, expect
          {'objective_value': -float('inf'), 'log_likelihood': float('inf'), 'likelihood': float('inf')})
     ])
 def test_constraints(minimizer, fit_kwargs, expected_par, expected_std, expected_obj, expected_got):
+    """
+    Tests the FitResults from fitting a simple model with Le as constraint
+    using several different minimizers and objectives.
+    """
+
+    model=Model({y: a * x ** b})
+    constraints = [
+        Le(a, b)
+    ]
```

Comment:
Could you add the constraint(s) to the parametrization so we can also test it with multiple constraints?
This test also needs a comment somewhere that the constraints must be symbolic

Reply:
That's difficult, because the expected results from the [other parametrization](https://github.com/tBuLi/symfit/pull/289/files/06027ef07c8781b5d0c0799304250054fedb101c..4dd37687f086d9219858268d07a8ded6a8f7d8d6#diff-bb0d67200a312294fe04c45b1925759cae6a59466f7c0302d619f9a035904961L31-L43) are defined by this specific constraint.

Reply:
Can't you add them to this one? https://github.com/tBuLi/symfit/pull/289/files/06027ef07c8781b5d0c0799304250054fedb101c..4dd37687f086d9219858268d07a8ded6a8f7d8d6#diff-bb0d67200a312294fe04c45b1925759cae6a59466f7c0302d619f9a035904961R68

Reply:
Oh..yes, we can do this with `Le`. (But it would be unreadable with CNM)

Reply:
Oh..yes, we can do this with `[Le(a,b)]`. (But it would be unreadable with CNM in my opinion)